supplyChainSimulation/MaterialPlanning.cs

```diff
@@ -76,59 +97,58 @@ public MaterialPlanning()
                     }
                 }
 
+                // customized for P1
+
```

Comment:
Der Code Part unter dem Kommentar `customized for P1` gehört nicht in den Block `if (initialInitMaterial == 0)`. Das führt zu nich geladenen Daten wenn man auf Seite `Materialplanung P1` **zurück** wechselt

Reply:
Die schließende Klammer `}` von Line 152 zu  Line 99 zu verschieben sollte das Problem lösen.